components/dashboard/src/AppNotifications.tsx

```diff
@@ -0,0 +1,107 @@
+/**
+ * Copyright (c) 2022 Gitpod GmbH. All rights reserved.
+ * Licensed under the GNU Affero General Public License (AGPL).
+ * See License-AGPL.txt in the project root for license information.
+ */
+
+import { useCallback, useEffect, useState } from "react";
+import Alert, { AlertType } from "./components/Alert";
+import { useFeatureFlag } from "./data/featureflag-query";
+
+const KEY_APP_DISMISSED_NOTIFICATIONS = "gitpod-app-notifications-dismissed";
+
+interface Notification {
+    id: string;
+    type: AlertType;
+    message: JSX.Element;
+}
+
+const SCHEDULED_DOWNTIME: Notification = {
+    id: "230924-scheduled-downtime",
+    type: "info",
+    message: (
+        <span className="text-md">
+            To improve the reliability of gitpod.io we are performing a scheduled maintenance this{" "}
```

Comment:
**nitpick**: Leaving a copy suggestion in case it helps, to match the design proposal in [EXP-657](https://linear.app/gitpod/issue/EXP-657/upgrade-page-placeholder-on-gitpod-cloud-during-migration#comment-28f69888) (internal). Other than that UX LGTM. 🏁 

```suggestion
            Gitpod will undergo scheduled system maintenance on{" "}
```